assets/styles/style.scss

```diff
@@ -247,3 +247,152 @@ $color-secondary: #c4c4c4;
     }
   }
 }
+
+// Single Product Gallery Bullet Nav
+.flex-control-paging {
+  display: block;
+  position: relative;
+  width: 100%;
+  height: 19px;
+  margin: 16px 0 0px;
+  padding: 0;
+  list-style: none;
+  text-align: center;
+
+  li {
+    position: relative;
+    display: inline-block;
+    cursor: pointer;
+    margin: 0;
+    padding: .25rem;
+    line-height: 0;
+    vertical-align: top;
+
+    a {
+      position: relative;
+      display: inline-block;
+      height: 11px !important;
+      width: 11px !important;
+      margin: 0;
+      padding: 0;
+      border-radius: 50%;
+      border: 2px solid $color-secondary;
+      background: 0 0;
+      background-color: transparent;
+      outline: 0;
+      line-height: 0;
+      vertical-align: top;
+      font-size: 0;
+      text-indent: -99999px;
+      color: transparent;
+      cursor: pointer;
+
+      &:focus,
+      &:hover {
+        outline: 0;
+
+        &:before {
+          opacity: .5;
+        }
+      }
+
+      &:hover {
+        &:before {
+          background-color: $color-secondary !important;
+        }
+        &:after {
+          content: "";
+          display: none !important;
+        }
+      }
+
+      &:before {
+        content: "";
+        position: absolute;
+        display: block;
+        width: 11px;
+        height: 11px;
+        top: -2px;
+        left: -2px;
+        border-radius: 50%;
+        color: $color-secondary;
+        transition: background .1s ease-out;
+      }
+
+      &.flex-active {
+        background-color: $color-secondary;
+      }
+    }
+  }
+}
+
+// Flickity WooCommerce Compatibility
+.woocommerce {
+  .product {
+    .flex-control-thumbs {
+      --thumbnail-count: 6;
+      --thumbnail-spacing-unit: 9px;
+      --thumbnail-slider-margin: 12px;
+
+      &.flickity {
+        display: block;
+        white-space: nowrap;
+        overflow: visible !important;
+        margin-left: var(--thumbnail-slider-margin) !important;
+        margin-right: var(--thumbnail-slider-margin) !important;
+
+        li {
+          float: none !important;
+          display: inline-block;
+          margin-right: var(--thumbnail-spacing-unit) !important;
+          margin-bottom: 0;
+          // Take the available width, subtract the necessary gaps between thumbnails and divide the remaining space evenly among thumbnails.
```

Comment:
If I am not wrong, we deal with inline comments in CSS files in a different way. We add a file block comment and there we add indexed comments. Something like:
```scss
/**
 * Gallerya layout and styling.
 *
 * 1. Take the available width, subtract the necessary gaps between thumbnails
 *    and divide the remaining space evenly among thumbnails.
 */
```
Then, on the affected line (349):
```scss
width: calc((100% - (var(--thumbnail-spacing-unit) * (var(--thumbnail-count) - 1))) / var(--thumbnail-count)) !important; // [1]
```
But probably @LucaPipolo or @colourgarden can make you some better suggestions about this.

Reply:
Yea if it's a longer comment (like this one) we'd usually pull it up to the top of the file and put `// [1]` as Joan suggests 👍